zh/monitor-a-tidb-cluster.md

```diff
@@ -1,21 +1,78 @@
 ---
-title: Kubernetes 与 TiDB 集群的监控与告警
-summary: 介绍如何在 Kubernetes 上部署 TiDB 集群监控。
-aliases: ['/docs-cn/tidb-in-kubernetes/dev/monitor-a-tidb-cluster/']
+title: TiDB 集群的监控与告警
+summary: 介绍如何监控 TiDB 集群。
+aliases: ['/docs-cn/tidb-in-kubernetes/dev/monitor-a-tidb-cluster/', '/docs-cn/tidb-in-kubernetes/dev/monitor-using-tidbmonitor/', '/zh/tidb-in-kubernetes/dev/monitor-using-tidbmonitor/']
 ---
 
-# Kubernetes 与 TiDB 集群的监控与告警
+# TiDB 集群的监控与告警
 
-基于 Kubernetes 环境部署的 TiDB 集群监控可以大体分为两个部分：对 **TiDB 集群本身**的监控、对 **Kubernetes 集群**及 **TiDB Operator** 的监控。
+本文介绍如何对通过 TiDB Operator 部署的 TiDB 集群进行监控及配置告警。
 
 ## TiDB 集群的监控
 
-TiDB 通过 Prometheus 和 Grafana 监控 TiDB 集群。在通过 TiDB Operator 创建新的 TiDB 集群时，可以参考[TidbMonitor 配置](monitor-using-tidbmonitor.md)，对于每个 TiDB 集群，创建、配置一套独立的监控系统，与 TiDB 集群运行在同一 Namespace，包括 Prometheus 和 Grafana 两个组件。
-
-可以在 `TidbMonitor` 中设置 `spec.persistent` 为 `true` 来持久化监控数据。开启此选项时应将 `spec.storageClassName` 设置为一个当前集群中已有的存储，并且此存储应当支持将数据持久化，否则会存在数据丢失的风险。
+TiDB 通过 Prometheus 和 Grafana 监控 TiDB 集群。在通过 TiDB Operator 创建新的 TiDB 集群时，可以对于每个 TiDB 集群，创建、配置一套独立的监控系统，与 TiDB 集群运行在同一 Namespace，包括 Prometheus 和 Grafana 两个组件。
 
 在 [TiDB 集群监控](https://pingcap.com/docs-cn/stable/deploy-monitoring-services/)中有一些监控系统配置的细节可供参考。
 
+### 持久化监控数据
+
+可以在 `TidbMonitor` 中设置 `spec.persistent` 为 `true` 来持久化监控数据。开启此选项时应将 `spec.storageClassName` 设置为一个当前集群中已有的存储，并且此存储应当支持将数据持久化，否则会存在数据丢失的风险。配置示例如下：
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: TidbMonitor
+metadata:
+  name: basic
+spec:
+  clusters:
+    - name: basic
+  persistent: true
+  storageClassName: ${storageClassName}
+  storage: 5G
+  prometheus:
+    baseImage: prom/prometheus
+    version: v2.18.1
+    service:
+      type: NodePort
+  grafana:
+    baseImage: grafana/grafana
+    version: 6.1.6
+    service:
+      type: NodePort
+  initializer:
+    baseImage: pingcap/tidb-monitor-initializer
+    version: v4.0.9
+  reloader:
+    baseImage: pingcap/tidb-monitor-reloader
+    version: v1.0.1
+  imagePullPolicy: IfNotPresent
+```
+
+你可以通过以下命令来确认 PVC 情况:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl get pvc -l app.kubernetes.io/instance=basic,app.kubernetes.io/component=monitor -n ${namespace}
+```
+
+```
+NAME            STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+basic-monitor   Bound    pvc-6db79253-cc9e-4730-bbba-ba987c29db6f   5G         RWO            standard       51s
+```
+
+### 自定义Prometheus 配置
+
+### 使用自定义配置文件
```

Comment:
The heading levels seem incorrect.

Reply:
fixed in https://github.com/pingcap/docs-tidb-operator/pull/978/commits/bb41f0d4e07b96c78e22f35b96f2e7ad2e2a012b.